### ISSUE TITLE
Ranking with data from Pandas Dataframe

### ISSUE THREAD

**Issue by Tejas-Nanaware** [tensorflow/ranking#310]
Hi,

Thanks for this amazing library. I was trying to implement `basic_ranking` [example](https://www.tensorflow.org/recommenders/examples/basic_ranking) with my own data from a data frame (which is basically numeric). I created a tf dataset using
```
training_dataset = (
    tf.data.Dataset.from_tensor_slices(
        (
            tf.cast(df[selected_features].values, tf.float32),
            tf.cast(df[target].values, tf.int32)
        )
    )
)
```

Then I created a Ranking model using
```
class RankingModel(tf.keras.Model):
    def __init__(self):
        super().__init__()
        
        self.ranks = tf.keras.Sequential([
            tf.keras.layers.Dense(128, activation='relu'),
            tf.keras.layers.Dense(128, activation='relu'),
            tf.keras.layers.Dense(128, activation='relu'),
            tf.keras.layers.Dense(1),
        ])
    def call(self, inputs):
        return self.ranks(inputs)

class RestaurantRankingModel(tfrs.models.Model):
    def __init__(self):
        super().__init__()
        self.ranking_model: tf.keras.Model = RankingModel()
        self.task: tf.keras.layers.Layer = tfrs.tasks.Ranking(
            loss = tf.keras.losses.MeanSquaredError(),
            metrics = [tf.keras.metrics.RootMeanSquaredError()]
        )
    
    def call(self, features) -> tf.Tensor:
        f, t = features
        return self.ranking_model(f)
    
    def compute_loss(self, features, training=False) -> tf.Tensor:
        f, t = features
        restaurant_predictions = self(f)
        return self.task(labels=t, predictions=restaurant_predictions)
```

But I keep getting error on unpacking. How can I unpack the tfdataset tensor to get the labels and features so that I can pass them to compute the loss.

I was hoping if you could assist me with this issue because I could not find any example anywhere that involved using tensorflow RankingModel using a dataframe.
